### PR TITLE
Fix filename in fuzzing corpus so it gets parsed during regression testing

### PR DESCRIPTION
**Pull request by fancycode**
Follow-up to #1204

### CODE REVIEW

**Comment by fancycode** [comment]
See https://github.com/strukturag/libheif/commit/29bc4c32514b1fc720b29dc1b0ac6cea4f7bf2b5#commitcomment-143843141

File: libheif/context.cc [hunk 1]
```diff
@@ -609,6 +609,17 @@ Error HeifContext::interpret_heif_file()
...
+          std::stringstream sstr;
+          sstr << "Image size " << width << "x" << height << " exceeds the maximum image size "
+                << m_maximum_image_size_limit << "\n";
+
+          return Error(heif_error_Memory_allocation_error,
+                        heif_suberror_Security_limit_exceeded,
+                        sstr.str());
+        }
+
         image->set_resolution(width, height);
```
Comment (farindk) [inline libheif/context.cc:623]: Simpler fix would be to set the argument type for `set_resolution()` to `uint32_t`. It is `uint32_t` internally anyway.

Comment (fancycode) [reply libheif/context.cc:623]: Ok, but it's returned as `int` in the getters and I didn't want to introduce any potential other errors by changing the type in various places.
https://github.com/strukturag/libheif/blob/e42b01f3b46eb02545b0ddeff4368d3dd3bdb430/libheif/context.h#L117-L119

Also it's returned as `int` in the public API:
https://github.com/strukturag/libheif/blob/e42b01f3b46eb02545b0ddeff4368d3dd3bdb430/libheif/api/libheif/heif.cc#L804
https://github.com/strukturag/libheif/blob/e42b01f3b46eb02545b0ddeff4368d3dd3bdb430/libheif/api/libheif/heif.cc#L815

Comment (farindk) [reply libheif/context.cc:623]: Ok, makes sense.

**Pull request merged**